src/main/configurations/Translate/Common/xsl/CreateDu02Response.xslt

```diff
@@ -48,7 +48,7 @@
                             <ZKN:versie><xsl:value-of select="$ZdsZaakDocumentInhoud/ZdsZaakDocumentInhoud/versie"/></ZKN:versie>
                         </xsl:if>
                         <xsl:if test="$ZdsZaakDocumentInhoud/ZdsZaakDocumentInhoud/status">
-                            <ZKN:status><xsl:value-of select="$ZdsZaakDocumentInhoud/ZdsZaakDocumentInhoud/status"/></ZKN:status>
+                            <ZKN:status><xsl:value-of select="$statusMap($ZdsZaakDocumentInhoud/ZdsZaakDocumentInhoud/status)"/></ZKN:status>
```

Comment:
The translation itself is correct, but it's in the wrong place. The response XSLT should only be concerned with the structure of the response message.

You can follow which adapters/soapActions result in a Du02Response and add this translation to the zgw to zds translations in them